File: Arcade/Intro/02_Edge_of_the_Ocean/04_adjacentElementsProduct/main.java
int adjacentElementsProduct(int[] inputArray) {
    int result = -1000;
    for (int i = 0; i < inputArray.length - 1; i++) {
        if (inputArray[i] * inputArray[i + 1] > result)
            result = inputArray[i] * inputArray[i + 1];
    }
    return result;
}

File: Arcade/Intro/02_Edge_of_the_Ocean/04_adjacentElementsProduct/main.ts
function adjacentElementsProduct(inputArray: number[]): number {
    let result: number = -1000;
    for (let i: number = 0; i < inputArray.length - 1; i++) {
        if (inputArray[i] * inputArray[i + 1] > result) {
            result = inputArray[i] * inputArray[i + 1];
        }
    }
    return result;
}
